hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/TokenHandlerHelper.java

```diff
@@ -251,6 +255,21 @@ public enum TokenValidations {
         PERMIT_PAUSED
     }
 
+    /**
+     * Enum to determine the type of validations to be performed on the token account relationship. If the token account
+     * relationship is allowed to be frozen or not
+     */
+    public enum TokenRelValidations {
+        /**
+         * Token account relationship should not be frozen
+         */
+        REQUIRE_NOT_FROZEN,
```

Comment:
This should be Freeze or Pause, or is pause logic somewhere else?

Reply:
Yes, and also by default it uses REQUIRE_NOT_FROZEN.
However, I think since we currently don't have a usecase to support this case, we could probably remove this enum and leave it required by default. ( Just so we don't add code thats not neccesary needed).

I could remove in the follow up PR, which will also change the protobuf branch

Reply:
Okay thanks